### PR TITLE
Integrate MAY-2022 ASB patches

### PR DESCRIPTION
**Pull request by rrsenapati1**
Tracked-On: OAM-101830
Signed-off-by: Senapati, Rasmiranjan <rasmiranjan.senapati@intel.com>

### CODE REVIEW

File: aosp_diff/preliminary/packages/services/Telecomm/01_0001-Handle-null-bindings-returned-from-ConnectionService-.patch [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+When a ConnectionService returns a null binding, immediately unbind from
+the ConnectionService and cancel any ongoing calls related to it.
+
+Bug: 211114016
+Test: Added new CTS test to verify auto unbind from null binding ConnectionService.
+Test: Manually tested using test app which implements null binding ConnectionService and verified via telecom log inspection that the service is unbound and the call is terminated.
+Change-Id: I0757557e66725dddfd871cd9857071a8749bd7ba
+(cherry picked from commit 410ce026004bb485c39afcc7d86e89d26ff1af94)
+Merged-In: I0757557e66725dddfd871cd9857071a8749bd7ba
+---
```
Comment (SaliniVenate) [inline aosp_diff/preliminary/packages/services/Telecomm/01_0001-Handle-null-bindings-returned-from-ConnectionService-.patch:15]: Please cross-check if this patch is already present as a part of previous tag. If yes please remove this patch and update the commit id details in the excel sheet you maintain.
I can see it as already applied in your EB stdio logs.
Project ./packages/providers/MediaProvider
                Already applied         04_0004-Avoid-apps-to-get-global-access-with-table-uri.patch

File: aosp_diff/preliminary/system/sepolicy/10_0010-0001-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Bug: 201645790
+Test: atest ListeningPortsTest
+Ignore-AOSP-First: Fix already in AOSP
+Change-Id: I0c66fb5e35cda3b1799cf003402e454d7a951e96
+(cherry picked from commit 1a6b37d838459b3d46cc06e31bc3c2bf5e7c387f)
+Merged-In:I0c66fb5e35cda3b1799cf003402e454d7a951e96
+---
+ prebuilts/api/30.0/private/adbd.te | 3 +++
+ private/adbd.te                    | 3 +++
+ 2 files changed, 6 insertions(+)
```
Comment (SaliniVenate) [inline aosp_diff/preliminary/system/sepolicy/10_0010-0001-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch:18]: This is also an already applied patch. Please cross-check and remove.
Project ./system/sepolicy
             Already applied         10_0010-0001-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch

Also the naming convention is wrong

**Review by SaliniVenate — CHANGES_REQUESTED** [review]

**Review by SaliniVenate — APPROVED** [review]
LGTM

**Review by shyjumon-n — APPROVED** [review]
Google security patch integration.
LGTM

**Pull request merged**